src/metaschema/readme.md

```diff
@@ -85,15 +92,15 @@ Nonetheless and with this in mind, understanding the mechanism by which the Meta
 
 Terminology note: "metaschema" is a common noun and there are many metaschema technologies (indeed almost any mature XML tag set has one), with different purposes, feature sets and capabilities. "Metaschema" (capitalized) is our peculiar homegrown metaschema technology and application.
 
-## The Overall Approach
+## The Approach
 
 A reduced, lightweight modeling language with certain specially-enforced constraints can be sketched such that multiple schemas constraining disparate formats, such as XML and JSON, can be produced from a single metaschema instance (a document written in Metaschema) deterministically, and thus designed and coordinated in parallel.
```

Comment:
@wendellpiez should we maybe be more specific on the use of XML for crafting the metaschemas themselves rather than alluding to any sort of new language which is what the sentence "a reduced, lightweight modeling language ..." kind of does? I know we don't want to detract from the design principles we've built, which themselves are new in this domain, but don't necessarily want to confuse folks when it comes to the syntax of the metaschemas.

Reply:
@anweiss good point. We could say "A reduced, lightweight XML-based modeling language" -- would that do the trick? I am not sure this is the place to explain the rationale(s) for the XML basis of the Metaschema itself. Or is it? We could also write a couple of sentences on that.

Reply:
I think simply adding "XML-based" will do the trick here. A better place to explain the rationale behind certain decisions such as the use of XML for the metaschema, etc might be in an FAQ of sorts, no?